circle.yml

```diff
@@ -0,0 +1,12 @@
+general:
+  build_dir: ./
+checkout:
+  post:
+    - curl --location 'https://nuget.org/nuget.exe' -o ./.nuget/NuGet.exe
+    - xbuild /p:Configuration=Release ./Docker.DotNet.sln
```

Comment:
ah, didn't know. I assumed checkout happens first. the default layout kinda imposes this.